llvm/include/llvm/Passes/MachinePassRegistry.def

```diff
@@ -192,6 +192,20 @@ MACHINE_FUNCTION_PASS("verify<machine-trace-metrics>", MachineTraceMetricsVerifi
 #define MACHINE_FUNCTION_PASS_WITH_PARAMS(NAME, CLASS, CREATE_PASS, PARSER,    \
                                           PARAMS)
 #endif
+
+MACHINE_FUNCTION_PASS_WITH_PARAMS(
+    "block-placement", "MachineBlockPlacementPass",
+    [](bool NoTailMerge) {
+      // Tail merging is enabled by default, so this option
+      // is to disable it.
+      return MachineBlockPlacementPass(!NoTailMerge);
```

Comment:
This inversion of the flag is confusion, it should be consistently one way or another